core/src/avm2/globals/flash/display/displayobject.rs

```diff
@@ -771,7 +772,7 @@ pub fn set_blend_mode<'gc>(
 
 fn new_rectangle<'gc>(
     activation: &mut Activation<'_, 'gc, '_>,
-    rectangle: Rectangle<Twips>,
+    rectangle: BoundingBox,
```

Comment:
Any special reason for this change? `Rectangle<Twips>` is convertible to `BoundingBox` and vice verse.

Reply:
(clarified on discord: there's no other-way conversion currently.)